sensing/gnss_poser/src/gnss_poser_core.cpp

```diff
@@ -135,18 +141,33 @@ void GNSSPoser::callbackNavSatFix(
     canGetCovariance(*nav_sat_fix_msg_ptr) ? nav_sat_fix_msg_ptr->position_covariance[4] : 10.0;
   gnss_base_pose_cov_msg.pose.covariance[6 * 2 + 2] =
     canGetCovariance(*nav_sat_fix_msg_ptr) ? nav_sat_fix_msg_ptr->position_covariance[8] : 10.0;
-  gnss_base_pose_cov_msg.pose.covariance[6 * 3 + 3] = 0.1;
-  gnss_base_pose_cov_msg.pose.covariance[6 * 4 + 4] = 0.1;
-  gnss_base_pose_cov_msg.pose.covariance[6 * 5 + 5] = 1.0;
+  gnss_base_pose_cov_msg.pose.covariance[6 * 3 + 3] =
+    std::pow(gnssOrientationRmse.rmse_rotation_x * deg2rad, 2);
```

Comment:
Since the units of `rmse_rotation_*` are radian, there is no need to do `deg2rad`.
https://github.com/autowarefoundation/autoware_msgs/blob/main/autoware_sensing_msgs/msg/GnssInsOrientation.msg#L10

Reply:
Fixed! https://github.com/autowarefoundation/autoware.universe/pull/1230/commits/c542142e6197bb1c1c118b3770391ca4fbbaf9ec